### ISSUE TITLE
The appium session creation fails as the the mFocusedApp parsed has a comma appended.

### ISSUE THREAD

**Issue by sandeepst** [appium/appium#6366]
## The problem
The appium session creation fails as the the mFocusedApp and the one expected are not matching.

The problem lies with the output of "dumpsys window windows" as shown below. The activity name is appended with a comma and an extra string isShadow:false. The issue is specific to le Tv 1s with EUI 5.5.012s. The android version is 5.0.2

```
sandeeps-MacBook-Air:junit sandeep$ /Users/sandeep/Library/Android/sdk/platform-tools/adb -s YSORNNTSQC9HN799 shell "dumpsys window windows" | grep ApiDemos
  Window #5 Window{94c238c u0 io.appium.android.apis/io.appium.android.apis.ApiDemos}:
    mToken=AppWindowToken{20fe217e token=Token{21878739 ActivityRecord{16425300 u0 io.appium.android.apis/.ApiDemos, isShadow:false t10}}}
    mRootToken=AppWindowToken{20fe217e token=Token{21878739 ActivityRecord{16425300 u0 io.appium.android.apis/.ApiDemos, isShadow:false t10}}}
    mAppToken=AppWindowToken{20fe217e token=Token{21878739 ActivityRecord{16425300 u0 io.appium.android.apis/.ApiDemos, isShadow:false t10}}}
    WindowStateAnimator{188c5815 io.appium.android.apis/io.appium.android.apis.ApiDemos}:
      mSurface=Surface(name=io.appium.android.apis/io.appium.android.apis.ApiDemos)
  mInteractingStatusBarWindow=Window{94c238c u0 io.appium.android.apis/io.appium.android.apis.ApiDemos}
  mCurrentFocus=Window{94c238c u0 io.appium.android.apis/io.appium.android.apis.ApiDemos}
  mFocusedApp=AppWindowToken{20fe217e token=Token{21878739 ActivityRecord{16425300 u0 io.appium.android.apis/.ApiDemos, isShadow:false t10}}}
```

Tried the same sample code with Emulator having version 5.0.1 and it worked.

## Environment
* Appium version: 1.4.13
* Desktop OS/version: Mac OSX El Capitan
* Mobile platform/version under test: Android/5.0.1
* Real device : Le tv 1s
* Appium CLI or Appium.app|exe: Appium.app
* App under Test : ApiDemos from appium sample code
* Binding : java client

## Link to Appium logs
https://gist.github.com/sandeepst/5cd350e71b2f51703cf2ec2e1d822941

## Code To Reproduce Issue [ Good To Have ]
The sample code is same as AndroidTests.java for ApiDemos-debug.apk under appium sample code.

Feel free to ask more information. Looking forward for quick help and I am thankful to you.

**Comment by imurchie** [comment]
Fixed in `appium-adb@2.6.1`. This fix will be in Appium 1.5.2.

**Comment by sandeepst** [comment]
Thank you very much Isaac. Good weekend.
